upgrade-2.x.sh

```diff
@@ -997,6 +983,27 @@ UUID=$(jq -r '.uuid' $CONFIGJSON)
 API_ENDPOINT=$(jq -r '.apiEndpoint' $CONFIGJSON)
 DELTA_ENDPOINT=$(jq -r '.deltaEndpoint' $CONFIGJSON)
 
+FETCHED_SLUG=$(curl -H "Authorization: Bearer ${APIKEY}" --silent --retry 5 \
+"${API_ENDPOINT}/v6/device?\$select=is_of__device_type&\$expand=is_of__device_type(\$select=slug)&\$filter=uuid%20eq%20%27${UUID}%27" 2>/dev/null \
+| jq '.d[0].is_of__device_type[0].slug'
```

Comment:
This needs to be `jq -r` ; otherwise the slug is set to `"device"` (notice the quotes!) instead of `device`.